sdks/java/io/hadoop/jdk1.8-tests/pom.xml

```diff
@@ -1,41 +1,38 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!--
-    Licensed to the Apache Software Foundation (ASF) under one or more
-    contributor license agreements.  See the NOTICE file distributed with
-    this work for additional information regarding copyright ownership.
-    The ASF licenses this file to You under the Apache License, Version 2.0
-    (the "License"); you may not use this file except in compliance with
-    the License.  You may obtain a copy of the License at
-
-       http://www.apache.org/licenses/LICENSE-2.0
-
-    Unless required by applicable law or agreed to in writing, software
-    distributed under the License is distributed on an "AS IS" BASIS,
-    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-    See the License for the specific language governing permissions and
-    limitations under the License.
--->
-<!-- The HifIO tests for Cassandra and Elasticsearch both work only with jdk1.8, but Beam's 
-enforcer rules require jdk1.7 and jdk1.8 support. This child module contains only those tests 
-and overrides the enforcer rules to allow 1.8 only behavior without making all of HifIO work only with jdk1.8. -->
+<!-- Licensed to the Apache Software Foundation (ASF) under one or more contributor 
```

Comment:
The formatting on the license here seems to be broken, can you please fix it back to the way it was?